### DISCUSSION TITLE
Moonraker Addition

### DISCUSSION THREAD

**Discussion by flyespresso** [xblax/flashforge_ad5m_klipper_mod#192] [Ideas]
Is it realistically possible to keep the stock build but add moonraker? Allowing OG controls onboard but if you want to have Fluid / Mainsail hosted off the device. Also lets you get 3rd party mobile apps.

I wouldn't mind putting it together but before I dive in, curious, could be a lightweight middle mod release :)

**Comment by consp** [comment]
Not a "complete" solution but you can disable fluidd/mainsail by renaming the scripts in `/etc/init.d` from "S[number][name]" to "K[number][name]", e.g. `S70fluidd` to `K70fluidd`. Or remove the scripts but that is more permanent.

**Reply by flyespresso** [reply]
Will functionality remain as normal then for the stock display and leveling functions? I can look into that if so.

**Reply by consp** [reply]
Misread the question, you want the stock app, then no not possible (see below).

**Comment by xblax** [comment]
I tried this out initially in the very early stages of the mod. Moonraker and the Stock app don't play together very nice, the app is unaware the Klipper state change by Moonraker and will crash. It's not worth putting more effort into this, there is absolutely no compatibility.

**Reply by consp** [reply]
Iirc it also sends stuff to the serial port directly which means it's nigh impossible to do it reliably.
